docs/configuration.md

```diff
@@ -51,24 +57,26 @@ targetCrdFilters:
 The autodiscovery mechanism will create a list of all CRD names in the cluster whose names have the suffix `group1.tnf.com` or `anydomain.com`, e.g. `crd1.group1.tnf.com` or `mycrd.mygroup.anydomain.com`.
 
 ## testTarget
+
 ### podsUnderTest / containersUnderTest
-The autodiscovery mechanism will attempt to identify the default network device and all the IP addresses of the pods it needs for network connectivity tests, though that information can be explicitly set using annotations if needed.
 
+The autodiscovery mechanism will attempt to identify the default network device and all the IP addresses of the pods it needs for network connectivity tests, though that information can be explicitly set using annotations if needed.
 
 #### Pod IPs
 
-* The `k8s.v1.cni.cncf.io/networks-status` annotation is checked and all IPs from it are used. This annotation is automatically managed in OpenShift but may not be present in K8s.
-* If it is not present, then only known IPs associated with the pod are used (the pod `.status.ips` field).
+- The `k8s.v1.cni.cncf.io/networks-status` annotation is checked and all IPs from it are used. This annotation is automatically managed in OpenShift but may not be present in K8s.
+- If it is not present, then only known IPs associated with the pod are used (the pod `.status.ips` field).
 
 #### Network Interfaces
 
-* The `k8s.v1.cni.cncf.io/networks-status` annotation is checked and the `interface` from the first entry found with `"default"=true` is used. This annotation is automatically managed in OpenShift but may not be present in K8s.
+- The `k8s.v1.cni.cncf.io/networks-status` annotation is checked and the `interface` from the first entry found with `“default”=true` is used. This annotation is automatically managed in OpenShift but may not be present in K8s.
 
-The label `test-network-function.com/skip_connectivity_tests` excludes pods from all connectivity tests. The label value is not important, only its presence.
-The label `test-network-function.com/skip_multus_connectivity_tests` excludes pods from [Multus](https://github.com/k8snetworkplumbingwg/multus-cni) connectivity tests. Tests on default interface are still done. The label value is not important, but its presence.
+The label `test-network-function.com/skip_connectivity_tests` excludes pods from all connectivity tests. The label value is trifling, only its presence.
```

Comment:
I'd rather this stay "not important".  "Trifling" makes no sense.

Reply:
`not important` triggers the linter. Other option?

Reply:
>does make build-catalog-md break any of this when ran
I guess it doesn't, I'm going to check.

Reply:
trivial is also an alternative.

Reply:
@sebrandon1 Is `The label value is trivial` good to go?

Reply:
Yes, `trivial` makes more sense to me.

Reply:
@sebrandon1 Fixed `make build-catalog-md`.